### ISSUE TITLE
Target "libvjson" does not exist (CMP0046)

### ISSUE THREAD

**Issue by steffen-AEI** [cvmfs-contrib/cvmfs-x509-helper#19]
```
-- Configuring done                                                                                                                                                                                                                                                                                                                                                        
CMake Warning (dev) at src/CMakeLists.txt:33 (add_dependencies):                                                                                                                                                                                                                                                                                                           
  Policy CMP0046 is not set: Error on non-existent dependency in                                                                                                                                                                                                                                                                                                           
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.                                                                                                                                                                                                                                                                                                 
  Use the cmake_policy command to set the policy and suppress this warning.                                                                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                                                                                                           
  The dependency target "libvjson" of target "cvmfs_x509_helper" does not                                                                                                                                                                                                                                                                                                  
  exist.                                                                                                                                                                                                                                                                                                                                                                   
This warning is for project developers.  Use -Wno-dev to suppress it.                                                                                                                                                                                                                                                                                                      
                                                                                                                                                                                                                                                                                                                                                                           
CMake Warning (dev) at src/CMakeLists.txt:34 (add_dependencies):                                                                                                                                                                                                                                                                                                           
  Policy CMP0046 is not set: Error on non-existent dependency in                                                                                                                                                                                                                                                                                                           
  add_dependencies.  Run "cmake --help-policy CMP0046" for policy details.                                                                                                                                                                                                                                                                                                 
  Use the cmake_policy command to set the policy and suppress this warning.                                                                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                                                                                                           
  The dependency target "libvjson" of target "cvmfs_scitoken_helper" does not                                                                                                                                                                                                                                                                                              
  exist.                                                                                                                                                                                                                                                                                                                                                                   
This warning is for project developers.  Use -Wno-dev to suppress it.                                                                                                                                                                                                                                                                                                      
                                                                                                                                                                                                                                                                                                                                                                           
-- Generating done                                                                                                                                                                                                                                                                                                                                                         
```

Is this warning benign or am I missing an essential build dependency? (I see this on Debian Jessie, Stretch, and Buster.)

**Comment by bbockelm** [comment]
@djw8605 - can you look at this today?  I'm stuck in meetings continuously and then headed on vacation for a week.

**Comment by djw8605** [comment]
@steffen-AEI I replicated and found the error.  It was a typo (libvjson vs vjson).  Fixed in #20.

**Comment by steffen-AEI** [comment]
Made this a patch in the source package (not touching the 2.0 tarball), and the warning is gone. Thanks!
